test/onigumo_cli_test.exs

```diff
@@ -60,6 +60,18 @@ defmodule OnigumoCLITest do
       end
     end
 
+    for switch <- ["-h", "--help"] do
+      test("run CLI with a #{inspect(switch)} switch") do
+        assert usage_message_printed?(fn -> Onigumo.CLI.main([unquote(switch)]) end)
+      end
+    end
+
+    for switches <- ["--help -C invalid", "-h --invalid"] do
+      test("run invalid combination of swiches #{inspect(switches)} ") do
```

Comment:
This may not be entirely exclusive use case for `-h`/`--help`, but rather a generic invalid combination of switches. I can imagine this being moved right after the existing invalid switch test, being different by providing a combination of a valid and an invalid argument.

I see even more user stories for the future:

* a valid and an invalid arguments provided at the same time,
* and an invalid combination of otherwise all valid arguments.

In our case, the latter is a combination of `-h`/`--help` and the component name.